.github/workflows/tests_linux_x86.yml

```diff
@@ -102,6 +103,7 @@ jobs:
       - name: Build and run unit tests for code coverage
         run: |
             cmake . -BBuildCov \
+              -DPLLGPU_ENABLE_MPI=OFF \
```

Comment:
As before, this is the default, right? Do we need it here?

Reply:
Removed! Done!